packages/select-menu/src/types.ts

```diff
@@ -89,6 +89,17 @@ export interface SelectMenuProps<T extends OptionBase> {
    */
   width?: number
   // minWidth?: number
+
+  /**
+   * Are items selectable
+   */
+  isSelectable?: boolean
```

Comment:
Summing up the logic, `closeOnSelect` implies that `isSelectable` as `false` - if we can't select something (in a lasting way) then we should hide checkboxes anyway. In other words: we can merge those two props into one, just name it properly.